sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,328 @@
+- Fixed `get_info<device::free_memory>` to check its aspect. [dfb75675bc0a]
+- Removed the workaround for release of auxiliary buffers. [fda9171fc218]
+- Enabled optimization for read-only buffer accesses from multiple devices in a context. [dddaf7f5d54d]
+- Removed old specialization constants extension and implementation. [e50ae05e28d3][833a9fe0a9cf]
+- Added README.md for the Unified Runtime directory. [453ffdcf41dd]
+- Improved `is_compatible` to check if specific target is defined with `-fsycl-targets` and change the result. [ef033238cfb5]
+- Improved quotient approximation for host `sycl::remquo`. [d335fdf4afa2]
+- Made `bit_cast.hpp` self-contained. [c899a93410c2]
+- Allowed accessor constructed with zero-size buffers. [0c0809590f37]
+- Improved resources recycling for the Level Zero backend by cleaning up events on in-order queue wait. [260182a1ad75]
+- Added `code_location` parameter to the rest of `sycl::queue` methods which allows to improve error reporting. [fdd609a5c724]
+- Enabled `xpti::node_create` signal emit for parallel_for that bypasses graph. [4eaaaa963ca2]
+- Improved the accuracy of host `sycl::cospi`. [37aa84b7afd5]
+- Promoted the return type changes of SYCL relational builtins that changed between SYCL 1.2.1 and SYCL 2020 out from the guard of `SYCL2020_CONFORMANT_APIS`. [35fe4a21e1cb]
+- Replaced usage of outdated memset OpenCL API `clEnqueueMemsetINTEL` with `clEnqueueMemFillINTEL`. [c5d0e1d4d9be]
+- Added memory pointer to XPTI memory allocation event metadata to allow more informative XPTI memory tracing. [01ffe65da562]
+- Implemented recycling of immediate command lists for queues in a context for the Level Zero backend. [2ddbf8c3b7a5]
+- Optimized `ext_oneapi_submit_barrier()` for in-order queue for the Level Zero backend. [c21fd476c78a]
+- Removed `sycl_ext_oneapi_extended_atomics` extension. [17ea6d10f1d5]
+- Removed `sycl_ext_oneapi_group_algorithms` extension. [8b9d2106af0d]
+- Removed unneeded backwards compatibility of 2023.2 make_queue and get_native. [c5f150a41b1d][5918e0c208a1]
+- Change the `SYCL_PI_LEVEL_ZERO_USM_RESIDENT` default to force device allocations only. [33874f76c59f]
+- Started to report false for `aspect::image` on all devices. [5cf0f7cf5c21]
+- Removed lambda parameter from "if_architecture_is" according to [`sycl_ext_oneapi_device_architecture`](doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc). [71f745bcda7d]
+- Throw for invalid reqd_work_group_size. [084c02797084]
+- Adjusted static restriction checks for block_2d APIs. [7902d6b35348]
+- Disallowed local accessor misuse according to SYCL 2020. [d7a5ec048368]
+- Implemented passing address space information to SPIR-V Joint Matrix Load/Store intrinsics. [854ab7eab5e4]
+- Enabled immediate command lists by default for the Level Zero backend. [9e5b2ed1ddc3]
+- Removed extra map lookup for eliminated kernel arguments during enqueue of cached kernels. [7a1fa6c38e03]
```

Comment:
I feel like this should either be something like "improved RT performance ...", or it should be removed. Tagging @sergey-semenov

Reply:
Fixed to "Improved SYCL RT performance by removing extra map lookup for eliminated kernel arguments during enqueue of cached kernels."